test/e2e/page-objects/flows/onboarding.flow.ts

```diff
@@ -6,48 +6,52 @@ import StartOnboardingPage from '../pages/onboarding/start-onboarding-page';
 import OnboardingCompletePage from '../pages/onboarding/onboarding-complete-page';
 
-export const importSRPOnboardingFlow = async (driver: Driver) => {
-  console.log('start import srp onboarding flow ');
+export const createNewWalletOnboardingFlow = async (driver: Driver) => {
+  console.log('start creat new wallet onboarding flow ');
   await driver.navigate();
   const startOnboardingPage = new StartOnboardingPage(driver);
   await startOnboardingPage.check_pageIsLoaded();
   await startOnboardingPage.checkTermsCheckbox();
-  await startOnboardingPage.clickImportWalletButton();
+  await startOnboardingPage.clickCreateWalletButton();
 
   const onboardingMetricsPage = new OnboardingMetricsPage(driver);
   await onboardingMetricsPage.check_pageIsLoaded();
   await onboardingMetricsPage.clickNoThanksButton();
 
-  const onboardingSrpPage = new OnboardingSrpPage(driver);
-  await onboardingSrpPage.check_pageIsLoaded();
-  await onboardingSrpPage.fillSrp();
-  await onboardingSrpPage.clickConfirmButton();
-
   const onboardingPasswordPage = new OnboardingPasswordPage(driver);
   await onboardingPasswordPage.check_pageIsLoaded();
-  await onboardingPasswordPage.createImportedWalletPassword();
+  await onboardingPasswordPage.createWalletPassword();
+
+  const secureWalletPage = new SecureWalletPage(driver);
+  await secureWalletPage.check_pageIsLoaded();
+  await secureWalletPage.revealAndConfirmSRP();
 };
 
-export const completeCreateNewWalletOnboardingFlow = async (driver: Driver) => {
-  console.log('start to complete create new wallet onboarding flow ');
+export const importSRPOnboardingFlow = async (driver: Driver) => {
+  console.log('start import srp onboarding flow ');
   await driver.navigate();
   const startOnboardingPage = new StartOnboardingPage(driver);
   await startOnboardingPage.check_pageIsLoaded();
   await startOnboardingPage.checkTermsCheckbox();
-  await startOnboardingPage.clickCreateWalletButton();
+  await startOnboardingPage.clickImportWalletButton();
 
   const onboardingMetricsPage = new OnboardingMetricsPage(driver);
   await onboardingMetricsPage.check_pageIsLoaded();
   await onboardingMetricsPage.clickNoThanksButton();
 
+  const onboardingSrpPage = new OnboardingSrpPage(driver);
+  await onboardingSrpPage.check_pageIsLoaded();
+  await onboardingSrpPage.fillSrp();
+  await onboardingSrpPage.clickConfirmButton();
+
   const onboardingPasswordPage = new OnboardingPasswordPage(driver);
   await onboardingPasswordPage.check_pageIsLoaded();
-  await onboardingPasswordPage.createWalletPassword();
-
-  const secureWalletPage = new SecureWalletPage(driver);
-  await secureWalletPage.check_pageIsLoaded();
-  await secureWalletPage.revealAndConfirmSRP();
+  await onboardingPasswordPage.createImportedWalletPassword();
+};
 
+export const completeCreateNewWalletOnboardingFlow = async (driver: Driver) => {
+  console.log('start to complete create new wallet onboarding flow ');
```

Comment:
```suggestion
  console.log('Starting to complete the new wallet onboarding flow');
```

Reply:
I am thinking this console statement could be removed. It looks like this in the console log 
![image](https://github.com/user-attachments/assets/712690bd-de10-4f84-a3c1-cf027b5c71b2)